service/ocean/providers/aws/launch_spec.go

```diff
@@ -180,6 +181,13 @@ type DeleteLaunchSpecInput struct {
 
 type DeleteLaunchSpecOutput struct{}
 
+type LaunchSpecStrategy struct {
```

Comment:
Could you please move it to line 143 (below the `EBS` struct) to keep all structs in order? Thanks.